spec/mem/tree_borrows/memory.md

```diff
@@ -65,41 +65,157 @@ impl TreeBorrowsFrameExtra {
 Here we define some helper methods to implement the memory interface.
 
 ```rust
+
+fn compute_nonfreeze_bytes(
+    cell_strategy: UnsafeCellStrategy,
+    layout_strategy: LayoutStrategy,
+    ptr_metadata: Option<PointerMeta<TreeBorrowsProvenance>>,
+    offset: Offset
+) -> List<(Offset, Offset)> {
+    match (cell_strategy, layout_strategy, ptr_metadata) {
+        (UnsafeCellStrategy::Sized { bytes }, ..) => bytes,
+        (UnsafeCellStrategy::Slice { element }, LayoutStrategy::Slice(..), Some(PointerMeta::ElementCount(count))) => {
```

Comment:
maybe the `UnsafeCellStrategy::Slice` needs the size of each repetition as well? @RalfJung

Reply:
I think either the unsafe cell info should just be part of `LayoutStrategy` or we have the `LayoutStrategy`  available here -- so there shouldn't be layout info in `UnsafeCellStrategy`.